relay-config/src/config.rs

```diff
@@ -169,7 +169,18 @@ trait ConfigObject: DeserializeOwned + Serialize {
 
     /// The full filename of the config file, including the file extension.
```

Comment:
Please update the documentation here too. It should reflect that the function returns the path to the first matching file, and falls back to the last extension if no files exist.